roc/version_test.go

```diff
@@ -0,0 +1,29 @@
+package roc
+
+import (
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestVersion(t *testing.T) {
+	v := Version()
+	require.NotZero(t, v.Library.Major+v.Library.Minor+v.Library.Patch)
+	require.NotZero(t, v.Bindings.Major+v.Bindings.Minor+v.Bindings.Patch)
+	require.Equal(t, v.Library.Major, v.Bindings.Major)
+	require.GreaterOrEqual(t, v.Bindings.Minor, v.Library.Minor)
+}
+
+func Test_bindingsVersion(t *testing.T) {
+	bvs := strings.SplitN(bindingsVersion, ".", 3)
```

Comment:
Thanks for the update! Does it pass locally for you? I'm not sure where does the var `bindingsVersion` come from, and on the CI it this test seems to be skipped. I haven't run it locally, let me try it later.

Reply:
> Does it pass locally for you?

yes

> I'm not sure where does the var bindingsVersion come from, and on the CI it this test seems to be skipped

it's from [latest release](https://github.com/roc-streaming/roc-go/releases), it's skipped because need to push changes with tag

Reply:
I remember injecting an env var during build [like so, with ldflags](https://stackoverflow.com/questions/68051479/how-to-go-build-ldflags-with-custom-environment-variable), didn't try it with libs though, should work with go build, right? Worth a try! Seems like [you can inject that stuff into packages as well, not just main](https://goenning.net/2017/01/25/adding-custom-data-go-binaries-compile-time/).

Reply:
Discussion moved to chat